addons/web/static/src/js/views/list/list_controller.js

```diff
@@ -203,6 +240,54 @@ var ListController = BasicController.extend({
             this._updatePaging(state);
         }
     },
+    /**
+     * @private
+     * @param {jQueryElement} $el
+     * @param {Object} node
+     */
+    _addOnHeaderButtonClickAction: function ($el, node) {
+        if (node.attrs.special || node.attrs.confirm || node.attrs.type) {
```

Comment:
do we need to handle special and confirm cases here? Don't we **always** have a type (thanks to rng or python validation)?